specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2020-05-01-preview/cpim.json

```diff
@@ -0,0 +1,521 @@
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "name": "Resource",
+            "in": "body",
+            "description": "Guest Usages Resource to be updated",
+            "schema": {
+              "$ref": "#/definitions/GuestUsagesResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GuestUsagesResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "GuestUsages_Delete",
+        "tags": [
+          "GuestUsages"
+        ],
+        "x-ms-examples": {
+          "GuestUsages_Delete": {
+            "$ref": "./examples/GuestUsagesDelete.json"
+          }
+        },
+        "summary": "Deletes a Guest Usages resource",
+        "description": "Deletes a Guest Usages resource for the Microsoft.AzureActiveDirectory resource provider",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK."
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "operationId": "GuestUsages_Get",
+        "tags": [
+          "GuestUsages"
+        ],
+        "x-ms-examples": {
+          "GuestUsages_Get": {
+            "$ref": "./examples/GuestUsagesGet.json"
+          }
+        },
+        "summary": "Gets a Guest Usages resource",
+        "description": "Gets a Guest Usages resource for the Microsoft.AzureActiveDirectory resource provider",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
```

Comment:
Use a more detailed description as above 'Guest Usage details successfully retrieved' for example

Reply:
done